templates/faq.html

```diff
@@ -48,7 +48,9 @@ <h2 id="find-the-current-dns-servers">Find the current DNS servers</h2>
       <p>To determine the current DNS servers used by the system run <code>systemd-resolve --status</code> (or <code>resolvectl</code> in 18.04 and higher) and look for the &#39;DNS Servers:&#39; entry to see what DNS server is used.</p>
       <h2 id="deconfigure-an-interface">Deconfigure an interface</h2>
       <p>To deconfigure an interface, remove the configuration for the device from the netplan .yaml file and run <code>sudo netplan apply</code>.</p>
-      <p>If the interface is not configured in a .yaml file in <code>/etc/netplan</code>, it will not be configured at boot. To remove addresses manually, a user can run <code>ip address del &lt;address&gt; dev &lt;interface&gt;</code>.</p>
+      <p><strong>Note:</strong> <code>netplan apply</code> will not remove virtual devices such as bridges and bonds that have been created, even if they are no longer described in the Netplan configuration. That is due to the fact that Netplan operates statelessly and is not aware of the previously defined virtual devices.</p>
+      <p>If the interface is not configured in a .yaml file in <code>/etc/netplan</code>, it will not be configured at boot. To remove an interface manually at runtime, a user can run <code>ip link del dev &lt;interface&gt;</code>.</p>
+      <p>Alternatively, a temporary backup of the YAML state in <code>/etc/netplan</code> can be created: <code>mkdir -p /tmp/old_state/etc &amp;&amp; cp -r /etc/netplan /tmp/old_state/etc/</code>. After creating such state, the interface can be removed from <code>/etc/netplan</code>, for example using: <code>rm /etc/netplan/interface-to-delete.yaml</code> and then be applied by calling: <code>netplan apply --state /tmp/old_state</code></p>
```

Comment:
This is not necessary, but I think it is easier to follow like this:
```suggestion
      <p>Alternatively, create a temporary backup of the YAML state in <code>/etc/netplan</code>: <code>mkdir -p /tmp/old_state/etc &amp;&amp; cp -r /etc/netplan /tmp/old_state/etc/</code>. After creating such state, remove the interface from <code>/etc/netplan</code> using, for example: <code>rm /etc/netplan/interface-to-delete.yaml</code>, and then apply the original state: <code>netplan apply --state /tmp/old_state</code></p>
```